hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyCommitRequest.java

```diff
@@ -301,6 +323,12 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
               exception, getOmRequest().getUserInfo()));
       processResult(commitKeyRequest, volumeName, bucketName, keyName,
           omMetrics, exception, omKeyInfo, result);
+    } else {
+      // Debug logging for HSync
+      if (LOG.isDebugEnabled()) {
```

Comment:
this check is redundant.

Reply:
I'm fine either way. I was following the same exercise applied elsewhere, mostly to avoid parameter evaluation I assume:

https://github.com/apache/ozone/blob/a66e43e2838f087229cb4b0fac1e1dc969ab80dd/hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/utils/BackgroundService.java#L103-L105

Reply:
There's no parameter evaluation cost if not in debug log level. https://www.slf4j.org/faq.html#logging_performance

Reply:
Yup.. I also wrote it this way in anticipation of other reviewers bugging me about adding this extra `if` check (been asked to do that before).

I will just remove it in this case.

Reply:
Done in d058d50b7f5dc07ebf26437ec5b14cb6e0f95a9b